packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -42,18 +44,58 @@ const metadata: Meta<TabsArgs> = {
     }
 };
 
+const simpleTabs: readonly TabArgs[] = [
```

Comment:
Sorry didn't state it more directly but can you make a similar change for anchor-tabs as well?

Reply:
![image](https://github.com/user-attachments/assets/10888508-ac0a-4759-996c-147941d7c234)